src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -17,9 +21,23 @@ class VeteranBenefitSummaryLetter extends React.Component {
   }
 
   render() {
+    const serviceInfo = this.props.benefitSummaryOptions.serviceInfo || [];
+    const militaryServiceRows = serviceInfo.map((service, index) => {
+      return (
+        <tr key={`service${index}`}>
+          <th scope="row" className="service-info">{(service.branch || '').toLowerCase()}</th>
+          <td className="service-info">
+            {characterOfServiceContent[(service.characterOfService).toLowerCase()]}
+            {/* _.get([(service.characterOfService).toLowerCase(), ''], characterOfServiceContent) */}
```

Comment:
I forget why my `_.get()` wasn't working, but thinking more generally, this is another case of weird/missing/unexpected data, i.e., an unexpected character of service type for which we have no content. However, backend schema validation should ensure this never happens. Just glanced a the benefit options schema and these values are strings but should probably be enums: https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/schemas/letter_beneficiary.json#L54